six/test/datadog_agent/datadog_agent_test.go

```diff
@@ -22,7 +22,12 @@ func TestMain(m *testing.M) {
 func TestGetVersion(t *testing.T) {
 	code := fmt.Sprintf(`
 	with open(r'%s', 'w') as f:
-		f.write(datadog_agent.get_version())
+		version = datadog_agent.get_version()
```

Comment:
Shouldn't the import be here?

Reply:
the code in `six/test/datadog_agent/datadog_agent.go` run all the pieces of python here. So importing it once there avoid duplicating the import multiple time here.